Thinktecture.Relay.Server/Http/ActionFilters/CheckModuleBindingConfigurationAttribute.cs

```diff
@@ -6,7 +6,7 @@
 using System.Web.Http.Filters;
 using Thinktecture.Relay.Server.Config;
 
-namespace Thinktecture.Relay.Server.Controller
+namespace Thinktecture.Relay.Server.Http.ActionFilters
```

Comment:
This should be named `Filters` instead, because the Frameworks namespace is `Filters`, too.